package.json

```diff
@@ -20,9 +20,11 @@
   "bugs": {
     "url": "https://github.com/shama/stylus-loader/issues"
   },
+  "peerDependencies": {
```

Comment:
`peerDependencies` are deprecated an impossible to maintain in some cases. We shouldn't support this. npm3's dedupe on install behaviour will have a similar but more maintainable effect allowing packages using stylus-loader to specify the stylus version they want as long as it matches stylus-loader's semver range. If that package uses a lower semver that stylus-loader doesn't support stylus-loader will use a version it can work with. If stylus-loader can support a lower semver we could increase the range. `0.x` seems too permissive is there an earlier version @timaschew you are using?